angr/analyses/decompiler/optimization_passes/eager_returns.py

```diff
@@ -42,15 +42,15 @@ def __init__(self, func, blocks_by_addr=None, blocks_by_addr_and_idx=None, graph
                  # settings
                  max_level=2,
                  min_indegree=2,
-                 region_identifier=None):
+                 **kwargs):
 
         super().__init__(func, blocks_by_addr=blocks_by_addr, blocks_by_addr_and_idx=blocks_by_addr_and_idx,
                          graph=graph)
 
         self.max_level = max_level
         self.min_indegree = min_indegree
         self.node_idx = count(start=node_idx_start)
-        self._ri = region_identifier
+        self._ri = kwargs.get("region_identifier", None)
```

Comment:
Why are you changing this line?

Reply:
Because I wanted to stop adding explicit parameters to the end of this optimization init. I thought this was cleaner for passing an arbitrary amount of data to it. I can, however, just make it all explicit if you like.

Reply:
Being explicit is always better than being implicit.

Reply:
https://peps.python.org/pep-0020/

Reply:
@ltfish dang, I just got the book thrown at me